Automattic/jetpack
# Protect: Integrate ScanReport

## Description
Adds `ScanReport` to the Home page in Protect

### Other information:


## Jetpack product discussion


- https://github.com/Automattic/jetpack-scan-team/issues/1521

## Does this pull request change what data or activity we track or use?


- No

## Testing instructions:






* Checkout this branch load Protect in JT
* Ensure you are redirected to the Home tab after passing setup
* Review the various states of the HomeStatCards error, scanning, idle, disabled
* Verify that the ScanReport appropriately reflects the state of the site
* Ensure no regressions are introduced

## Screenshots
![Screen Shot 2024-12-02 at 13 59 11](https://github.com/user-attachments/assets/2d2f130d-8256-46ff-a817-09e7dcbdeb2c)
